site/en/docs/extensions/mv3/tut_websockets/index.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Using WebSockets in Service Workers'
+seoTitle: 'How to use WebSockets in Chrome Extension Service Workers'
+date: 2023-07-04
+description: 'Step-by-step instructions on how to connect to a WebSocket in your Chrome Extension.'
+---
+
+This tutorial demonstrates how to connect to a WebSocket in your Chrome Extension's service worker. You can find a [working example on Github](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/tutorial.websockets).
+
+## Background
+
+Starting with Chrome M116, extension service workers get full support for WebSockets. Previously, a service worker could become inactive if the WebSocket connection was active for more than 30s and no other extension events occurred at that time, which could result in connection loss and the inability to receive messages.
```

Comment:
It does not look like the M notation is used for chrome versions elsewhere on the site.

Reply:
Remove